src/pages/documentation/resources/alerts.md

```diff
@@ -0,0 +1,7 @@
+<Variant platform="outdated-version" componentClass="InlineNestedAlert" repeat="1"/>
+
+<InlineNestedAlert variant="warning" header="false" iconPosition="left">
```

Comment:
I think this still shows as a grey exclamation, can we use something like variant="error" to increase visibility?

Reply:
Changes look good, can we also include this alert on the privacy resources?